tests/matcher.test.ts

```diff
@@ -88,6 +90,12 @@ describe("Route matcher", function () {
           "/foo/baz" => {
             "pattern": "/foo/baz",
           },
+          "/without-trailing" => {
+            "pattern": "/without-trailing",
+          },
+          "/with-trailing" => {
```

Comment:
(this is valid -- currently we _lose_ this information in routing table as trailing slash by default is to be ignored)